docs/how-to/encryption.md

```diff
@@ -0,0 +1,125 @@
+# Managing Encryption
+
+The Apache Cassandra charm has encryption enabled by default.
+All units within a cluster share the same CA certificate file, but each unit has a distinct private key.
+
+This charm implements the **Requirer** side of the [`tls-certificates/v1`](https://github.com/canonical/charm-relation-interfaces/blob/main/interfaces/tls_certificates/v1/README.md) relation. Therefore, any charm implementing the **Provider** side can be used.
+To enable TLS encryption, you must first deploy a TLS certificates Provider charm.
+
+---
+
+## Deploy a TLS Provider Charm
+
+For testing purposes, you can use the `self-signed-certificates` charm.
+However, this setup is **not recommended** for production clusters.
+
+Deploy the `self-signed-certificates` charm with:
+
+```shell
+# Deploy the TLS charm
+juju deploy self-signed-certificates --channel=edge
+
+# Configure the CA common name
+juju config self-signed-certificates ca-common-name="Test CA"
+```
```

Comment:
Suggestion: Remove the comments from the code block and split the commands with normal text, introducing the second command.